lambda/app/src/utils/helpers.ts

```diff
@@ -70,6 +70,8 @@ export const processRequest = (data: any, isMerged: boolean, isAdmin: boolean) =
 
   if (data.protocol === 'saml') data.authType = 'browser-login';
 
+  if (data.teamId !== null && data.usesTeam === false && data.projectLead === true) data.teamId = null;
```

Comment:
Do we need to set the projectLead to null as well?

Reply:
@thegentlemanphysicist projectLead is a boolean and it can be false or true. The default value of it is false.